autosklearn/automl.py

```diff
@@ -690,11 +691,18 @@ def fit(
         # The metric must exist as of this point
         # It can be provided in the constructor, or automatically
         # defined in the estimator fit call
-        if self._metric is None:
-            raise ValueError("No metric given.")
-        if not isinstance(self._metric, Scorer):
+        if self._metrics is None:
```

Comment:
This cannot happen, right? Because self._metrics is set in l614

Reply:
Correct. I just removed this check.